index.html

```diff
@@ -48,8 +50,238 @@ <h1 class="main-name">Erik Stoupignan</h1>
+                </div>
+                <!-- See this project button -->
+                <div class="box align-button box6">
+                    <a href="#">
+                        <div class="buttom-arrow">
+                            <p>See this project
+                            <img class="arrow" src="/assets/211621_c_right_arrow_icon.png" alt="Right arrow">
+                            </p>
+                        </div>
+                    </a>
+                </div>
+            </div>
+
+            <!-- Section  #5-->
+            <div class="container">
+                <img class="box1" src="/assets/Rectangle21(3).png" alt="Decoration photo">
+                <div class="box box2">
+                    <p class="subtitle">Project names goes here</p>
+                </div>
+                <!-- Button's Code -->
+                <div class="box align box3">
+                    <p class="code-button">HTML/CSS</p>
+                </div>
+                <div class="box align box4">
+                    <p class="code-button">Ruby on Rails</p>
+                </div>
+                <div class="box align box5">
+                    <p class="code-button">JavaScript</p>
+                </div>
+                <!-- See this project button -->
+                <div class="box align-button box6">
+                    <a href="#">
+                        <div class="buttom-arrow">
+                            <p>See this project
+                            <img class="arrow" src="/assets/211621_c_right_arrow_icon.png" alt="Right arrow">
+                            </p>
+                        </div>
+                    </a>
+                </div>
+            </div>
+
+            <!-- Section  #6-->
+            <div class="container">
+                <img class="box1" src="/assets/Rectangle21(4).png" alt="Decoration photo">
+                <div class="box box2">
+                    <p class="subtitle">Project names goes here</p>
+                </div>
+                <!-- Button's Code -->
+                <div class="box align box3">
+                    <p class="code-button">HTML/CSS</p>
+                </div>
+                <div class="box align box4">
+                    <p class="code-button">Ruby on Rails</p>
+                </div>
+                <div class="box align box5">
+                    <p class="code-button">JavaScript</p>
+                </div>
+                <!-- See this project button -->
+                <div class="box align-button box6">
+                    <a href="#">
+                        <div class="buttom-arrow">
+                            <p>See this project
+                            <img class="arrow" src="/assets/211621_c_right_arrow_icon.png" alt="Right arrow">
+                            </p>
+                        </div>
+                    </a>
+                </div>
+            </div>
+
+    </div>
+
+    <!-- About me -->
+    <div class="about">
```

Comment:
- [ ] Nice working on your about section, kindly make the container of the about section a `section` tag for better use of HTML semantics

Reply:
Ok, perfect. I already fix. Thank you!